AISKU/Tests/ApplicationInsightsDeprecatedTests.ts

```diff
@@ -2,6 +2,10 @@ import { IAppInsightsDeprecated } from "../src/ApplicationInsightsDeprecated";
 import { ApplicationInsightsContainer } from "../src/ApplicationInsightsContainer";
 import { Snippet } from "../src/Initialization";
 import { Sender } from "@microsoft/applicationinsights-channel-js";
+import { SinonSpy } from "sinon";
```

Comment:
I've updated the AISKU test folder, so it now creates a single full test *.js file which includes all of the referenced files, previously it required that the test.html file "pre-loaded" sinon.

This makes it a *lot* easier to upgrade the version of sinon being used.